main/acle.md

```diff
@@ -7156,9 +7158,29 @@ Assuming `float64_t` data, the C version of the code above would be:
   p3 = svrdffr();
 ```
 
+### SVE reinterpret intrinsics
+
+The `svreinterpret` intrinsics for vector types and vector tuple types take the form
+
+``` c
+svreinterpret_type0[_type1][_xN]
```

Comment:
```suggestion
svreinterpret_type0[_type1_xN]
```
Elsewhere, we've tended to remove `][`, as in @CarolineConcatto 's comment.  The idea is to avoid giving the impression that each part can be dropped individually.